swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition=None, fabric="Unknown", age=None):
+        super().__init__(id, condition, age)
+        self.fabric = fabric
+
+    def __str__(self):
+        return f"An object of type {self.__class__.__name__} with id {self.id}. "\
+        f"It is made from {self.fabric} fabric."
```

Comment:
Notice that each `__str__` method for `Clothing`, `Decor`, and `Electronic` all start with **"An object of type [CLASSNAME] with id [ID]."**. Inside of `Item` we can have a `__str__` method that print could return the part mentioned above and then pair that with the `__str__` inside the child classes (`Clothing`, `Decor`, and `Electronic`). It could something like this: 

```python
class Item: 
...
	def  __str__(self):
		return  f"An object of type {self.get_category()}" \
		f" with id {self.id}."

class Clothing(Item):
...
	def  __str__(self):
		return  f"{super().__str__()} It is made " \
		f"from {self.fabric} fabric."
```